pkgs/tools/networking/stunnel/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname = "stunnel";
-  version = "5.64";
+  version = "5.65";
 
   src = fetchurl {
     url    = "https://www.stunnel.org/downloads/${pname}-${version}.tar.gz";
-    sha256 = "sha256-7r5T7RFrpDsueGdisMK5FRHnt0hXrUdlgk5xmeb6+IM=";
-    # please use the contents of "https://www.stunnel.org/downloads/${name}.tar.gz.sha256",
+    sha256 = "60c500063bd1feff2877f5726e38278c086f96c178f03f09d264a2012d6bf7fc";
+    # please use the contents of "https://www.stunnel.org/downloads/${pname}-${version}.tar.gz.sha256",
     # not the output of `nix-prefetch-url`
```

Comment:
FYI well, yeah but no. If nix-prefetch-url shows a different hash that will not work

Reply:
I think that was the point of this advice?  If the server's .sha256 file and nix-prefetch-url produce different results, something has gone wrong and one shouldn't just charge ahead with whatever nix-prefetch-url says, but stop to figure out what the trouble is.

My thought in updating this advice was to follow the name -> pname+version change (46420bbaa3f8f79ce7b9ee68e98eba1f7bce2db6) so that the suggested action was easier to perform by copy/pasting the pname and version lines into a shell to set the variables so then one could copy/paste the advice's URL and have it expand correctly.

I've reverted the change to the advice comment to keep this PR narrowly focused on the version bump that's required to fix the build for non-tarball-cache-users.